src/main/java/com/cloudbees/jenkins/support/filter/ContentFilters.java

```diff
@@ -59,8 +59,8 @@ public boolean isEnabled() {
     }
 
     @DataBoundSetter
-    public void setEnabled(boolean enabled) {
-        this.enabled = enabled;
+    public void setEnabled(Boolean enabled) {
```

Comment:
It's odd you have to box the boolean to `Boolean` for this.

Reply:
Right now, it seems like you're right -- I remember hitting something involving DBC that got upset, but w/ my current code so far it seems there is a way for it to work.